### ISSUE TITLE
 impossible to insert a picture already available on the website (Internal Image)

### ISSUE THREAD

**Issue by serge73** [plone/plone.app.content#77]
When upgrading plone.app.content to version 3.0.18 or above, it is impossible to insert a picture already available on the website (Internal Image) to the Page.

The picture file is not recorded in the drop-down menu.

![2016-03-15_0-51-44](https://cloud.githubusercontent.com/assets/13430305/13758026/0f6e00a8-ea49-11e5-9b7b-1774335cc525.jpg)

**Comment by davilima6** [comment]
Which browser is this? Any console errors? I couldn't reproduce with Firefox/Ubuntu and p.a.content 3.0.20 (5.3-pending). Is this only after an upgrade? Does it work for you in vanilla 5.3-pending?

**Comment by serge73** [comment]
I tested p.a.content 3.0.20 and 3.0.17: 
Server is Debian i386 latest stable
Client:
a) Windows 10 (two different PC). Latest IE, chrome, Opera
b) Debian latest Opera

Firefox (win) and Icewiasel (Debian) do not load the TinyMCE toolbar. Check out quickly was not able.

Will try to test using Ubuntu on VM Ware. What version of Ubuntu you used?
What did you do when upgrading from 5.0.2 to 5.0.3?

**Comment by davilima6** [comment]
I didn't try upgrading. Is this an upgrade-only issue?

I tested on Ubuntu 15.10. And now I also tested with Windows browsers, newest stable Chrome and FF, and it's ok too.

**Comment by serge73** [comment]
On Plone 5.0.2 under the above conditions p.a.c 3.0.20 does not work, as with 5.0.3.

Plone 5.0.2 + 3.0.17 without any problems.

I need to install a fresh OS and Plone to make a video.

**Comment by vangheem** [comment]
@serge73 Are there any errors in the browser console?

It's also not clear if you tried this on a fresh install or if this is only happening on your upgraded instance?

**Comment by serge73** [comment]
Unexpectedly found another solution for 3.0.20: installing site language to English!

The problem appears when using Russian, Polish, African and probably any other non-English

**Comment by serge73** [comment]
Please set the Russian or any other non-English and test issue:
![2016-03-22_22-31-21](https://cloud.githubusercontent.com/assets/13430305/13961707/88912172-f07f-11e5-91c5-145530343781.jpg)
Console if you installed the Russian:
![2016-03-22_22-34-18](https://cloud.githubusercontent.com/assets/13430305/13961711/8c722d86-f07f-11e5-8295-7a4993401f9b.jpg)
Console if you installed the English:
![2016-03-22_22-39-14](https://cloud.githubusercontent.com/assets/13430305/13961714/901e22d2-f07f-11e5-93ba-01ad2fa822f6.jpg)

**Comment by serge73** [comment]
In Plone version 5.0.3, the problem is solved similarly to language change.

**Comment by davilima6** [comment]
Indeed I tested it without upgrading but choosing pt-br when adding the site and couldn't even select the added image. Thus I also couldn't insert a picture in Tiny.

**Comment by vangheem** [comment]
Looks like vocabulary content results, like portal_type, are getting translated and screwing things up:

https://github.com/plone/plone.app.content/commit/f16726643e9d3b0368b8958b193bdddf2a9a641d
and
https://github.com/plone/plone.app.content/commit/92e974961f1bf09fc8b00434be20779c8fd18b3d

seem to be responsible.

\cc @terapyon

**Comment by serge73** [comment]
Wanted to try to install the English only for the admin.
But the site does not change the language remains a language which is default for the site.
A global problem with the use of languages other than English.

Advise where to place the information for specialists, what would it drew attention.

![2016-04-05_10-12-02](https://cloud.githubusercontent.com/assets/13430305/14272047/867b619c-fb18-11e5-9030-d301c93a1462.jpg)

**Comment by vangheem** [comment]
It should be an easy fix in plone.app.content. portal_type should not be getting translated. ping @terapyon

**Comment by thet** [comment]
I have a similar issue with a translated portal_type which I'm going to fix here: https://github.com/plone/plone.app.content/pull/83/commits/aab7f4ce9297bf14bc10fa1296d7765498dde93d
I think I can take care of this too.

**Comment by jensens** [comment]
``portal_type`` must never be translated. It is expected as a stable identifier for the type, the id in the types tool. The Type (iirc this is always the ``title`` of the FTI) is what you want to display and translate.

**Comment by thet** [comment]
Fixed here: https://github.com/plone/plone.app.content/pull/83/commits/1a451373f8ac477a4267febaaeb739e37b5c0eac

> -  Added most notably `portal_type`, `review_state` and `Subject` but also `exclude_from_nav`, `is_folderish`, `last_comment_date`, `meta_type` and `total_comments` to ``BaseVocabularyView`` ``translate_ignored`` list.
>  Some of them are necessary for frontend logic and others cannot be translated.
>  Fixes https://github.com/plone/plone.app.content/issues/77

**Comment by davilima6** [comment]
@serge73, thank you for the report. Would you mind opening it as a separate issue?

**Comment by serge73** [comment]
davilima6, no problem.
Where it to write? Products.CMFPlone or another product?

**Comment by terapyon** [comment]
Sorry too late, I'll check this issue.

**Comment by 1letter** [comment]
I have the same issue in my Plone 5.0.4 installation. when i set the language to "German", i could not insert a internal image, i can see no js-errors in the console. But when i set the language to "English", then all is fine. Only the js error with the wrong Filename en_gb.js vs. en_EN.js is show in the console.

**Comment by thet** [comment]
This should already be fixed in latest plone.app.content

**Comment by 1letter** [comment]
Oh, and what ist the latest version? 3.0.20 via pypi wich is installed on my system or is this the master-branch this git-repro? I used the normal tar-installer-file of plone. i'm confused, sorry.

**Comment by davilima6** [comment]
@1letter, I believe @thet was referring to 3.0.21 unreleased as you can see here: https://github.com/plone/plone.app.content/blob/master/CHANGES.rst

@thet, could you please make a release? There seems to be multiple issues open for this bug.

**Comment by thet** [comment]
@davilima6 I don't have pypi rights for that.

I think the next release is worth a minor version bump - I raised the version to 3.1.
@plone/release-team can you please make a release?

**Comment by gforcada** [comment]
plone.app.content 3.1 is released: https://pypi.python.org/pypi/plone.app.content/3.1 :tada:
